examples/examples.json

```diff
@@ -140,6 +155,11 @@
                     "title": "Hurricane Harvey",
                     "desc": "Rainfall totals from Hurricane Harvey aggregated to counties. The animation is based on total rainfall. Counties with more rain draw first to lower rainfall totals. The opacity is based off population density (total pop / area). The brighter polygons have higher populations.",
                     "file": "advanced/hurricane-harvey.html"
+                },
+                {
+                    "title": "Torque controls",
+                    "desc": "Control torque animations",
+                    "file": "advanced/torque-controls.html"
```

Comment:
I disagree, I understand this can be seen as a form of interaction, but I think "interactivity" is for reading back properties

Reply:
Oh, I just saw this one. I understand your point, but for users not knowing the technology behind as strongly as you do, this makes wayyy more sense in interactivity than here. Also because the examples in Advanced are truly advanced, in terms of story, or styling, or complexity... this map is super simple in comparison (so... not really advanced 😛, just a simple map with controls).

Reply:
I'll buy that... :crying_cat_face:

Reply:
🙂